plugins/maven/src/main/java/org/wildfly/swarm/plugin/maven/StopMojo.java

```diff
@@ -59,8 +62,35 @@ public void executeSpecific() throws MojoExecutionException, MojoFailureExceptio
         if (tmp != null && tmp.exists()) {
             tmp.delete();
         }
+
+        Path tmpDir = new File(System.getProperty("java.io.tmpdir")).toPath();
```

Comment:
This looks a bit dangerous. Also there's `TempFileManager.TMPDIR_PROPERTY`, not sure how relevant it is here.

Reply:
Those are different properties, and I think would only cover certain cases.